src/diffusers/pipelines/onnx_utils.py

```diff
@@ -190,7 +198,12 @@ def _from_pretrained(
             )
             kwargs["model_save_dir"] = Path(model_cache_path).parent
             kwargs["latest_model_name"] = Path(model_cache_path).name
-            model = OnnxRuntimeModel.load_model(model_cache_path, provider=provider, sess_options=sess_options)
+            model = OnnxRuntimeModel.load_model(
+                model_cache_path,
+                provider=provider,
+                sess_options=sess_options,
+                provider_options=kwargs.get("provider_options"),
```

Comment:
```suggestion
                provider_options=kwargs.pop("provider_options"),
```
this is intended for `load_model` not cls() no? if that's the case let's pop so it won't get pass on